wordpress-mobile/gutenberg-mobile
# Brings ENTER > split in iOS up to date with Android.

Updates the `react-native-aztec` dependency so that ENTER > split is up to date with Android.

### Testing:

- Run the app in iOS.
- Make sure ENTER > split works as in Android.
